### PR TITLE
feat: Make the query page query the right dataset

### PR DESCRIPTION
**Pull request by manuzope**
So that when we go to `http://localhost:1218/outcomes/query` and click `Run`, we query the outcomes dataset. And similarly, for other datasets too.

### CODE REVIEW

File: snuba/templates/query.html [hunk 1]
```diff
@@ -2,7 +2,10 @@
 {% block content %}
     <script>
     function run() {
-      fetch('/query', {
+      dataset = window.location.pathname.split("/").filter(function(p){ return p !== "" })[0]
```
Comment (fpacifici) [inline snuba/templates/query.html]: does this still work on events if you open /query without passing a dataset in the URL ?

Comment (manuzope) [reply snuba/templates/query.html]: Yah. It redirects to `/events/query`.

Comment (manuzope) [reply snuba/templates/query.html]: I had more conditionals before that would check for that case but given that we're redirecting, this felt cleaner.

**Review by tkaemming — COMMENTED** [review]
Oops, when I changed this recently I made the (obviously incorrect, in retrospect) assumption that this was just always querying the URL of the current page, not explicitly providing a path.

File: snuba/templates/query.html (hunk 1 above)
Comment (tkaemming) [reply snuba/templates/query.html]: Tried to add a better comment earlier, but internet: can't the URL passed here just be the pathname itself? I don't think any URL parsing needs to happen here.

Comment (manuzope) [reply snuba/templates/query.html]: Ooo that's even better. Change coming!

**Review by tkaemming — APPROVED** [review]
👌

**Pull request merged**